tests/README.md

```diff
@@ -19,7 +19,7 @@ python3 -m venv venv
 # Activate virtual environment
 source venv/bin/activate
 # Install necessary modules
-pip install requests pytest
+pip install -U requests pytest
```

Comment:
These should probably be managed as dev dependencies with poetry. Using the latest versions like this will eventually cause the tests to break.

Reply:
Nice. i'll make a ticket for this

Reply:
#31